src/schematic/traits.rs

```diff
@@ -29,10 +29,10 @@ pub const INGRESS: &str = "ingress";
 pub const AUTOSCALER: &str = "auto-scaler";
 pub const MANUAL_SCALER: &str = "manual-scaler";
 pub const VOLUME_MOUNTER: &str = "volume-mounter";
-pub const INGRESS_V1ALPHA1: &str = "ingress.core.oam.dev/v1alpha1";
-pub const AUTOSCALER_V1ALPHA1: &str = "auto-scaler.core.oam.dev/v1alpha1";
-pub const MANUAL_SCALER_V1ALPHA1: &str = "manual-scaler.core.oam.dev/v1alpha1";
-pub const VOLUME_MOUNTER_V1ALPHA1: &str = "volume-mounter.core.oam.dev/v1alpha1";
+pub const INGRESS_V1ALPHA1: &str = "ingress";
```

Comment:
>INGRESS_V1ALPHA1 [](start = 10, length = 16)

The earlier plan was to hold making the change for backward compatibility till v0.2 is released. I'm not sure of that state now. @suhuruli might know about it.

Reply:
Not sure what's plan here?